scripts/Inelastic/Direct/detpackmap.py

```diff
@@ -0,0 +1,10 @@
+def sequoia(name):
+    name = name.strip().upper()
+    packs = (['B%s' % i for i in range(1, 37+1)]
```

Comment:
[PEP 3101](https://www.python.org/dev/peps/pep-3101/) suggests replacing the `%` formatting with `.format`